src/manifest.c

```diff
@@ -235,7 +235,10 @@ static struct manifest *manifest_from_file(int version, char *component, bool he
 		}
 		if (latest && strncmp(component, "MoM", 3) == 0) {
 			if (strncmp(line, "actions:", 8) == 0) {
-				post_update_action = strdup(c);
+				post_update_actions = list_prepend_data(post_update_actions, strdup(c));
+				if (!post_update_actions->data) {
+					abort();
```

Comment:
If this was part of the main update code path I'd agree with abort, but I think it would be better in this case to just print a loud warning saying we couldn't parse/read the action, and that an update or verify should be run, instead of aborting. We want the update to always proceed since it's critical, but executing the action is arguably not critical imho - considering the current state of things :)